rdtho2525/whats-cookin
# Feature/full recipe card

## Is this a fix or a feature? ⬇️

Feature

## What is the change? ⬇️

All code in scripts.js and styles.css is new. In addition, all HTML outside of what was provided in the starter repo is new.

## Where should the reviewer start? ⬇️

Full review of scripts.js, styles.css, and index.html.

## How should this be tested? ⬇️

- Should be able to filter recipes by multiple tags
- Should be able to search recipes by their name or ingredients
- Full recipe card
  - should appear when an individual recipe card is clicked on
  - should display that individual card's image, name, cost, ingredients, instructions
  - should exit this view upon clicking the X button in top right
